src/components/security_manager/test/security_query_matcher.cc

```diff
@@ -99,6 +99,7 @@ ::testing::AssertionResult QueryHeader_EQ(
     const ::security_manager::SecurityQuery::QueryHeader& q1,
     const ::security_manager::SecurityQuery::QueryHeader& q2);
 
+// cppcheck-suppress unusedFunction //Used in unit tests
```

Comment:
is predeclaration above necessary? I think it can be removed.

also can we wrap this function in `#ifdef BUILD_TESTS`?

Reply:
No, this predeclaration is not necessary and was deleted in 9f1a645791e598169044d384c2a131addb74c075
Also `#ifdef BUILD_TESTS` was added in 9f1a645791e598169044d384c2a131addb74c075